spec/game_spec.rb

```diff
@@ -2,28 +2,160 @@
 
 RSpec.describe Game do
     let(:game) { Game.new }
-    describe '#score' do
+
+    describe '#ninth_frame?' do
+        it 'returns true if turn is 8' do
+            game.turn = 8
+            expect(game.ninth_frame?).to be true
+        end
```

Comment:
```suggestion
    describe '#ninth_frame?' do
        context 'when it is the 8th turn' do
            it 'returns true' do
                game.turn = 8
                expect(game.ninth_frame?).to be true
            end
        end
```

So this reads as "When it is the 8th turn it returns true". More like documentation and "English-y"